src/extract.ts

```diff
@@ -133,6 +133,39 @@ export interface PytestBenchmarkJson {
     version: string;
 }
 
+type JuliaBenchmarkTrialEstimate = [
+    string,
+    {
+        params: [
+            string,
+            {
+                seconds: number;
+                samples: number;
+                evals: number;
+                overhead: number;
+                gctrial: boolean;
+                gcsample: boolean;
+                time_tolerance: number;
+                memory_tolerance: number;
+            },
+        ];
+        time: number;
+        gctime: number;
+        memory: number;
+        allocs: number;
+    },
+];
+
+type JuliaBenchmarkGroup = [
+    string,
+    {
+        data: Record<string, JuliaBenchmarkGroup | JuliaBenchmarkTrialEstimate>;
+        tags: string[];
+    },
+];
+
+type JuliaBenchmarkJson = [string, JuliaBenchmarkGroup[]];
```

Comment:
Looking at the output the first item in a tuple should be `Object`
```suggestion
type JuliaBenchmarkJson = [Object, JuliaBenchmarkGroup[]];
```